### DISCUSSION TITLE
Downtime for https://searxng.diadz.de

### DISCUSSION THREAD

**Discussion by datakraut** [searxng/searx-instances#295] [Report instance downtime]
The instance will be unavailable starting in the next couple of hours due to server migration. I cannot give an estimated downtime window as migrating docker containers is a bit more complex, especially if you do not have physical access to the storage drive (cloud server), but it should not take longer than 24h.

**Comment by unixfox** [comment]
Thank you for keeping us updated. For now we will keep your instance in the list if the downtime is not too long.

**Comment by datakraut** [comment]
The migration has been completed, and the instance is back online, it is possible that the instance may have a few hiccups in the next 24h due to the new server but for now everything seems to be working as expected.
